### PR TITLE
Added image prefetch

### PR DESCRIPTION
**Pull request by mrayor**
- Added image prefetch
- Changed base url
- Fixed signup bug

### CODE REVIEW

File: App.jsx [hunk 1]
```diff
@@ -24,17 +28,27 @@ export default class App extends Component {
   state = {
     isReady: false
   };
-  async componentDidMount() {
```
Comment (ekediala) [inline App.jsx]: `useEffect` works perfectly for this. `useEffect( () => {}, []) === componentDidMount`.

Comment (mrayor) [reply App.jsx]: The app component is essentially a class-based component.

File: src/components/verification/components/VerifyOtp.jsx [hunk 2]
```diff
@@ -89,14 +91,16 @@ const VerifyOtp = () => {
     });
   };
   handleSubmit = () => {
+    if (!state.twilio) return;
```
Comment (ekediala) [inline src/components/verification/components/VerifyOtp.jsx]: What does this mean for the user? If `!state.twilio `is true, how does the user know what to do next?

Comment (mrayor) [reply src/components/verification/components/VerifyOtp.jsx]: If there is an error from Twilio, `state.twillo` will not exist and if the user submits anyway, the the next line returns `undefined`. So if there is an error, the user attends to the error.

**Comment by mrayor** [comment]
> k

There is nothing wrong with using class and functional components.

File: App.jsx (hunk 1 above)
Comment (ekediala) [reply App.jsx]: Why? Thought we were using React Hooks.

File: src/components/verification/components/VerifyOtp.jsx (hunk 2 above)
Comment (ekediala) [reply src/components/verification/components/VerifyOtp.jsx]: OK. In essence, the error would be on screen for the user if `state.twilio` is `undefined`.

**Comment by mrayor** [comment]
No, that doesn’t happen.

**Pull request merged**